egs/europarl_st/SRT/lcma_srt/datamodule.py

```
@@ -0,0 +1,580 @@
+
+    @lru_cache()
+    def gigaspeech_dev_cuts(self) -> CutSet:
+        logging.info("About to get Gigaspeech dev cuts")
+        return load_manifest_lazy(self.args.manifest_dir / "cuts_DEV.jsonl.gz")
+
+    @lru_cache()
+    def dev_cuts(self) -> CutSet:
+        logging.info("About to get dev cuts")
+        return load_manifest_lazy(self.args.manifest_dir / self.args.dev_name)
+
+    @lru_cache()
```

Ruff (0.15.20)

[warning] Use of `functools.lru_cache` or `functools.cache` on methods can lead to memory leaks

(B019)